mudb/src/client.ts

```diff
@@ -2,12 +2,10 @@ import { MuSocket } from './socket';
 import { MuMessageInterface, MuAnyMessageTable, MuAnyProtocolSchema, MuProtocolFactory } from './protocol';
 
 export class MuRemoteServer<Schema extends MuAnyMessageTable> {
-    public message:MuMessageInterface<Schema>['userAPI'];
-    public sendRaw:(bytes:Uint8Array|string, unreliable?:boolean) => void;
+    public message!:MuMessageInterface<Schema>['userAPI'];
+    public sendRaw!:(bytes:Uint8Array|string, unreliable?:boolean) => void;
```

Comment:
Add a constructor and set these variables when `MuRemoteServer` is instatiated.

Reply:
too many changes need to be made, and not necessarily making the code better. really not sure if it's worth it.